src/python_testing/TC_DEM_2_10.py

```diff
@@ -162,7 +162,7 @@ async def test_TC_DEM_2_10(self):
         self.step("4")
         sub_handler = ClusterAttributeChangeAccumulator(Clusters.DeviceEnergyManagement)
         await sub_handler.start(self.default_controller, self.dut_node_id,
-                                self.matter_test_config.endpoint,
+                                self.get_endpoint(),
```

Comment:
Should we consistently provide a default in all cases?

Reply:
This matches the current behaviour, so I think for now let's push this through then talk about forced endpoint setting in a followup. That probably needs a bigger PSA / consultation.